client/my-sites/theme/main.jsx

```diff
@@ -346,21 +347,35 @@ const ThemeSheet = React.createClass( {
 	},
 
 	renderSupportTab() {
-		if ( this.props.isCurrentUserPaid ) {
+		const { isCurrentUserPaid, isJetpack, forumUrl, isWpcomTheme, isLoggedIn } = this.props;
+		let buttonCount = 1;
+
+		if ( isLoggedIn ) {
 			return (
 				<div>
-					{ this.renderContactUsCard( true ) }
-					{ this.renderThemeForumCard() }
-					{ this.renderCssSupportCard() }
+					{ isCurrentUserPaid && ! isJetpack &&
+						this.renderSupportContactUsCard( buttonCount++ ) }
+					{ forumUrl && isWpcomTheme &&
```

Comment:
Do we need the `isWpcomTheme` check here? Currently I see nothing at all for custom themes on jetpack sites:

<img width="638" alt="screen shot 2017-02-13 at 12 19 30" src="https://cloud.githubusercontent.com/assets/7767559/22883170/c49b974c-f1e6-11e6-9b49-442ffa0b63f6.png">